### PR TITLE
Create and test selector-no-attribute

### PR DESCRIPTION
**Pull request by m-allanson**

### CODE REVIEW

File: src/rules/selector-no-attribute/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# selector-no-type
+
+Disallow attribute selectors.
+
+```css
+    a[rel="external"] {}
+/**  ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑
```
Comment (jeddy3) [inline src/rules/selector-no-attribute/README.md]: Can we lose the `a` and just have one arrow in the middle?

File: src/rules/selector-no-attribute/__tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+testRule(null, tr => {
+  tr.ok("foo {}")
+  tr.ok(".bar {}")
+  tr.ok("foo .bar {}")
+
+  tr.notOk("[foo] {}", messages.rejected)
+  tr.notOk("a[rel=\"external\"] {}", messages.rejected)
+  tr.notOk("a, .foo[type=\"text\"] {}", messages.rejected)
+  tr.notOk("a > [foo] {}", messages.rejected)
```
Comment (jeddy3) [inline src/rules/selector-no-attribute/__tests__/index.js]: Can we add a single quotation marks reject test please?

`tr.notOk("a[rel='external'] {}", messages.rejected)`

**Comment by jeddy3** [comment]
Thanks for this! :) Just two very minor things, otherwise it looks great.

Congrats on adding rule 80! :tada:  :)

**Comment by davidtheclark** [comment]
Looks good.

**Comment by jeddy3** [comment]
Awesome thanks!

**Comment by m-allanson** [comment]
:+1:

**Pull request merged**